src/components/categories/Categories.jsx

```diff
@@ -0,0 +1,48 @@
+import "./Categories.css";
+import axios from "axios";
+import { useEffect } from "react";
+import { useAPI } from "../../context/api-context";
+
+export function Categories() {
+  const { APIState, APIDispatch } = useAPI();
+  const { data, error, loading } = APIState;
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await axios.get("/api/categories");
```

Comment:
I would suggest you destructure the data here 
```const {data} = await axios.get("/api/categories");```

Reply:
Yes, will do it. Thanks!